.github/workflows/create-gitHub-prerelease-after-fast-forward-check.yml

```diff
@@ -0,0 +1,48 @@
+name: Create GitHub prerelease after fast-forward check
+on: workflow_dispatch
+
+jobs:
+  prerelease:
+    name: prerelease
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Access to the incoming release
+        uses: actions/checkout@v3
+        with:
+          repository: amzn/ion-java
+          path: current_release
+
+      - name: Build ion-java
+        run: |
+          cd current_release
+          git submodule init
+          git submodule update
+          mvn clean install
```

Comment:
> This causes the tests to be run. I'd be okay skipping that, since this workflow will only be run when previous workflows have confirmed the tests pass. Let's use `mvn clean install -DskipTests`.

Thanks for the suggestion, I'll update this.